media-gfx/exiv2/files/exiv2-0.25-install-dirs.patch

```diff
@@ -0,0 +1,22 @@
+--- a/config/CMakeChecks.txt	2015-07-19 14:17:22.857046105 +0200
```

Comment:
I cant find a reference that this is mandatory, so please force such stuff on own your own.
https://devmanual.gentoo.org/ebuild-writing/misc-files/patches/index.html